app/controllers/kiosk_controller.rb

```diff
@@ -24,7 +22,7 @@ def show
 
   def reload_kiosk?(kiosk)
     return false if kiosk.restart_at.nil? || kiosk.restart_at_active.nil?
-    kiosk.restart_at_active && kiosk.restart_at < DateTime.now
+    kiosk.restart_at_active && kiosk.restart_at < DateTime.now && kiosk.restart_at > 1.minutes.ago
```

Comment:
I think we should make a method in the Kiosk model that does the restart evaluations.. something like

```
def should_restart?
  restart_at_active && restart_at < DateTime.now && restart_at > 1.minutes.ago
end
```
That way you call this method in the controller and view `kiosk.should_restart?`

Reply:
This concept helps to keep the hardcoded `1.minutes.ago` in a single place